docs/deployment.md

```diff
@@ -0,0 +1,34 @@
+# Architecture-Overview
+
+![architecture-overview](images/ci_cd_github_big_picture_public.drawio.png)
+_The picture shows the architecture overview. The steps are explaind in the following passages_
+
+# Source Repo
+
+The top left from the architecture-overview is explaint in [link](./workflows.md).
+
+# Helm Chart
+
+![architekture-external-helm-chart](images/external-helm-chart.drawio.png)
+
+The top right from the architecture-overview is explaint in the following.
+
+We provide a sample chart. https://github.com/it-at-m/helm-charts/tree/main/charts/sps-sample, which consists subcharts. Each subchart is a different module for example frontend, backend, eai. Each module was created with `helm create` and we made some adaptions. The Chart.yml includes the external dependency of the [RefArch Api Gateway Helm Chart](https://github.com/it-at-m/helm-charts/tree/main/charts/refarch-gateway). The main config is done in the values.yml of the parent chart. In the sections refarch-gateway, frontend, backend you can configure the subcharts.
+
+# Deployment
+
+The bottom part from the architecture-overview is expliant in the following.
+
+At the end of a sprint a developer builds a release in the source code project. It needs to be done for each folder frontend/backend/eai separately. Then the devops engineer needs to update the chart-version and app-version of the helm chart in the Chart.yaml.
```

Comment:
Ich würde diesen Teil wie folgt aufbauen:

1. Was nutzen wir intern (mit Verlinkung auf die offinziellen Links):
    1. Die Image Registry Quai, in das die Docker Images der Projekte synchronisiert werden
    2. GitLab zur Ausführung der internen Pipelines (IaC) und Konfiguration der Anwendungen
    3. OpenShift für den Betrieb der Anwendungen
 2. Dann wie es funktioniert
     1. Autorollout für latest images (nur in nicht-produktive Umgebungen)
     2. Versionierung für deployment in produktive oder produktionsnahe Umgebungen:
        1. über WF maven-build-release (setzen der Versionsnummer)
        2. Manuelles Triggern der internen Pipeline